pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -1,30 +1,26 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // HyperConvergedSpec defines the desired state of HyperConverged
 // +k8s:openapi-gen=true
 type HyperConvergedSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	KubeVirtImagePullPolicy      string `json:"KubeVirtImagePullPolicy,omitempty"`
-	CDIImagePullPolicy           string `json:"CDIImagePullPolicy,omitempty"`
-	NetworkAddonsImagePullPolicy string `json:"NetworkAddonsImagePullPolicy,omitempty"`
+	// Version of all the components under the HCO
+	Version string `json:"Version"`
```

Comment:
Is the idea here that each of the components is going to have a version, imagePullPolicy, and containerRegistry (maybe under a different name) configurable in their CR Spec?

Reply:
Yes, each component operator should express these variables in some way on their CR.  https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22#discussion_r273484482